core/worker/src/main/java/com/cognifide/aet/worker/drivers/FirefoxWebDriverFactory.java

```diff
@@ -68,6 +76,10 @@
   @Property(name = LOG_FILE_PATH, label = "Path to firefox error log", value = DEFAULT_FIREFOX_ERROR_LOG_FILE_PATH)
   private String logFilePath;
 
+  @Property(name = SELENIUM_GRID_URL, label = "Url to selenium grid hub", value = DEFAULT_SELENIUM_GRID_URL)
```

Comment:
Please update the documentation and explain there the logic that is behind switching from local ff driver to selenium grid one.
For now it looks that the logic is, if user defines `seleniumGridUrl` parameter value, Selenium Grid is used.
Is that fair, that this one is preferred over `path` parameter?
This logic should be documented.